GeoUtils/raster_tools.py

```diff
@@ -70,6 +74,36 @@ def __str__(self):
         """ Provide string of information about Raster. """
         return self.info()
 
+    def _read_attrs(self, attrs=None):
+        # Copy most used attributes/methods
+        if attrs is None:
+            self._saved_attrs = default_attrs
+            attrs = default_attrs
+        else:
+            if isinstance(attrs, str):
+                attrs = [attrs]
+            for attr in default_attrs:
```

Comment:
IIRC, this forces all default_attrs to be added to the Raster even when a list of them is provided by the user. Not sure that this is desirable behaviour?

Reply:
That is correct. The reason being, we are assuming that a number of default attributes are in place for other functions, such as info() - if they're not set, info() won't work - and crop() may not either as currently implemented.

Reply:
OK, I see. In which case it would be worth noting this in the docstring for the class and/or __init__ - something along the lines of, 'all the defaults are always loaded, if you want to add any other attributes then list them here'